src/components/Company/AssignSignatory/InviteSignatory/InviteSignatoryForm.tsx

```diff
@@ -70,8 +68,7 @@ export const InviteSignatoryForm = () => {
             onChange: handleEmailChange,
           }}
```

Comment:
Same suggestion here, you could just move this `onChange` to the root component api instead of inputProps

Reply:
Fixed